src/App.tsx

```diff
@@ -1,16 +1,23 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [initialMovies, setInitialMovies] = useState<Movie[]>(moviesFromServer);
```

Comment:
```suggestion
  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
```
it's the list of movies you display, there are also added movies, so the `initial` word is redundant